### ISSUE TITLE
Optimize the accessibility of the application

### ISSUE THREAD

**Issue by anmi2004** [T8RIN/ImageToolbox#773]
Dear developer, I am a blind user from China. I am using your program and many of the features are very useful and beneficial to me.

However, there are some problems with the accessibility of the program. Many of the buttons are not labeled for accessibility, which makes it impossible to read them properly with a screen reader. While it is true that blind people have lower needs when it comes to image processing, there are still many features within the software that are useful to me, especially when it comes to filling out forms and such that require images.

Optimizing this software for accessibility shouldn't take too much of your time, you just need to follow Google's Accessibility Development Guidelines to make changes to its button properties. For more information, please refer to the Android developer's accessibility guidelines page.

**Comment by T8RIN** [comment]
Thanks for the suggestion, i will look :)
